src/file_list/file_list.js

```diff
@@ -1,53 +1,53 @@
-import tmp from 'tmp';
-import jetpack from 'fs-jetpack';
-import scanner from 'receipt-scanner';
-import Select from './select';
-import Util from './util';
+import tmp from 'tmp'
+import jetpack from 'fs-jetpack'
+import scanner from 'receipt-scanner'
+import Select from './select'
+import Util from './util'
 
 export class FileList {
-  constructor(elId) {
-    this.elId = elId,
-    this.files = [],
-    this.fileIndex = -1,
-    this.selectedFiles = [],
-    this.Select = new Select(this),
-    this.Util = new Util(this),
-    this.maxConcurrent = 2;
+  constructor (elId) {
+    this.elId = elId
+    this.files = []
+    this.fileIndex = -1
+    this.selectedFiles = []
+    this.Select = new Select(this)
+    this.Util = new Util(this)
+    this.maxConcurrent = 2
   }
 
-  el() {
-    return document.getElementById(this.elId);
+  el () {
+    return document.getElementById(this.elId)
   }
 
-  addFiles(files) {
-    for(var i = 0; i < files.length; ++i) {
-      var extractedFiles = this.Util.extractFiles(files[i]);
-      for(var j = 0; j < extractedFiles.length; ++j) {
-        this.addFile(extractedFiles[j].name, extractedFiles[j].path, extractedFiles[j].size, extractedFiles[j].type);
+  addFiles (files) {
+    for (var i = 0; i < files.length; ++i) {
+      var extractedFiles = this.Util.extractFiles(files[i])
+      for (var j = 0; j < extractedFiles.length; ++j) {
+        this.addFile(extractedFiles[j].name, extractedFiles[j].path, extractedFiles[j].size, extractedFiles[j].type)
       }
     }
   }
 
-  getFileIndexForIndex(index) {
-    return this.files.findIndex(function(f) { return f.index === index; });
+  getFileIndexForIndex (index) {
+    return this.files.findIndex(function (f) { return f.index === index })
   }
 
-  createSmoothPercentProgressionInterval(index) {
+  createSmoothPercentProgressionInterval (index) {
     var interval = new Util.Interval(() => {
```

Comment:
Here's is the usage of ECMA6 function definition. It should be kept consistent throughout, and it might be good to just switch all the code over to ECMA6 (?) This should be done in another PR though.